lib/project_types/node/cli.rb

```diff
@@ -0,0 +1,15 @@
+module Node
```

Comment:
I love this template for each project type this is perfect. Easy to read and will be great for OS users as well.